mobile/src/navigations/onboarding-navigation/components/steps/steps.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+    type DrawerContentComponentProps,
+    DrawerContentScrollView,
+} from '@react-navigation/drawer';
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { Step } from '../components';
+import { styles } from './styles';
+
+const Steps: React.FC<DrawerContentComponentProps> = (props) => {
+    const { navigation, state } = props;
+
+    return (
+        <DrawerContentScrollView
+            {...props}
+            style={[globalStyles.m25, globalStyles.flex1]}
+        >
+            <Pressable
+                onPress={(): void => {
+                    navigation.closeDrawer();
+                }}
```

Comment:
I know that it should not like this, but when Im commiting changes an error appears. `@typescript-eslint/unbound-method`

Reply:
Do you mean something like this?
```
<Pressable
    onPress={navigation.closeDrawer}
    style={styles.button}
>
```

Reply:
Yes